src/DashboardService.Models.Db/DbTask.cs

```diff
@@ -6,18 +6,18 @@ namespace LT.DigitalOffice.DashboardService.Models.Db;
 public class DbTask
 {
   public const string ToTable = "Tasks";
-  
+
   public Guid Id { get; set; }
   public Guid GroupId { get; set; }
   public Guid? TaskTypeId { get; set; }
   public Guid? PriorityId { get; set; }
 
   public string Name { get; set; }
   public string Content { get; set; }
-  
+
   public DateTime CreatedAtUtc { get; set; }
-  public DateTime? DeadLineAtUtc { get; set; }
-  
+  public DateTime? DeadlineAtUtc { get; set; }
+
   public DbGroup Group { get; set; }
   public DbTaskType? TaskType { get; set; }
   public DbPriority? Priority { get; set; }
```

Comment:
Why do you use here nullable?

Reply:
why not?

Reply:
Because you can do a task without an end date

Reply:
we did it for case when we need do delete priority and not to delete task because of FK. in that case we will have task with null priority

Reply:
Simply, in our project it is not customary to make reference types nullable.

Reply:
And now a lot of warnings 
![image](https://github.com/LT-Students/DashboardService/assets/43515136/72c47a06-e44c-4ac2-863c-f562c08e3b58)